.circleci/config.yml

```diff
@@ -71,6 +71,7 @@ jobs:
       - run:
           name: Run tests with coverage
           command: |
+            GORACE=halt_on_error=1
```

Comment:
Is there no `export ` needed here? I don't really understand the difference to be honest.

Reply:
> halt_on_error (default 0): Controls whether the program exits after reporting first data race.

https://golang.org/doc/articles/race_detector.html#Options

Reply:
I was referring to the (missing?) `export` of `GORACE`, not the content of it.
See https://stackoverflow.com/a/1158231/2013738